build/build.sh

```diff
@@ -33,11 +33,19 @@ pandoc --verbose \
   --bibliography=$BIBLIOGRAPHY_PATH \
   --csl=$CSL_PATH \
   --metadata link-citations=true \
-  --mathjax \
   --include-after-body=build/themes/default.html \
```

Comment:
This is probably more a question for @dhimmel:
I am little bothered by the fact that all these html files are actually JavaScript embedded in `<script>..</script>`. Wouldn't it be better to use `--include-in-header` option to include the JavaScript files directly?

Reply:
I'm not sure what you mean by this.

The `--include-in-header` option of pandoc basically just copies and pastes text data into the `.html` output, verbatim. It doesn't hotlink things eg `<script src="http://googleapis.com/jquery.min.js"></script>`. I think the maintainer of pandoc actually did add a feature (at Daniel's request) to allow a URL reference in the `--include` parameters, but that still just fetches the resource at the URL and pastes it verbatim into the document, same as normal.

Also, some of our plugin files necessitate multiple `<script>` tags for configuration, for example Mathjax:

```
<script type="text/x-mathjax-config">
    // plugin configuration
</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML">
    // link to or code from MathJax or similar plugin
</script>
```
Even though our plugins will only ever need one `<script>` tag, I wanted to keep all the plugin files consistent in format and extension.

Daniel and I had talked about possibly moving our plugins to a hosted repo and hotlinking them in the `--include` parameters instead of having them directly in the manuscript repo, but they would still take the same form: `.html` files with `<script>` tags.

Reply:
I am not an expert of `pandoc`. If `--include-in-header` option doesn't help, then we don't any other options. :-P

Reply:
It's weird to put javascript in an `.html` file, for sure. I don't know of any precedence for doing so. But I think we're in weirdly specific territory here, and I think under the circumstances it's the cleanest solution.